### PR TITLE
Add translation for Starting an open source project

### PR DESCRIPTION
**Pull request by sunny0826**

### CODE REVIEW

File: content/en/guides/starting.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Open source projects provide freedom to collaborate – even among competitors i
...
+The decision to release or create a new open source project depends on your circumstances. Your company should first achieve a certain level of open source mastery by using open source software and contributing to existing projects. This is because consuming can teach you how to leverage external projects and developers to build your products. And participation can bring more fluency in the conventions and culture of open source communities. (See our guides on [Using Open Source Code](https://todogroup.org/resources/guides/using-open-source-code/) and [Participating in Open Source Communities](https://todogroup.org/resources/guides/participating-in-open-source-communities/)) But once you have achieved open source fluency, the best time to start launching your own open source projects is simply “early” and “often.”
```
Comment (node) [inline content/en/guides/starting.md:32]: Good work.  
We'd better split this PR , one for fixing the URL issues in English version to sync these updates to our guys in English , and another for Chinese version translation .

**Review by node — CHANGES_REQUESTED** [review]
Split into two PRs.

File: content/en/guides/starting.md (hunk 1 above)
Comment (sunny0826) [reply content/en/guides/starting.md]: done

**Review by node — APPROVED** [review]
LGTM

**Comment by node** [comment]
We can not  rebase and merge now.  
It seems you need to update this branch firstly because of the later PR #411 has been merged.

**Pull request merged**